src/managers/KeybindManager.cpp

```diff
@@ -1048,14 +1048,20 @@ SWorkspaceIDName getWorkspaceToChangeFromArgs(std::string args, PHLWORKSPACE PCU
         return {WORKSPACE_NOT_CHANGED, ""};
     }
 
-    const auto ID = PCURRENTWORKSPACE->m_iID;
-    if (const auto PWORKSPACETOCHANGETO = g_pCompositor->getWorkspaceByID(PPREVWS.id); PWORKSPACETOCHANGETO) {
-        if (PER_MON && PCURRENTWORKSPACE->m_iMonitorID != PWORKSPACETOCHANGETO->m_iMonitorID)
-            return {WORKSPACE_NOT_CHANGED, ""};
+    const auto ID                   = PCURRENTWORKSPACE->m_iID;
```

Comment:
unintentional?

Reply:
This change is from b7f42a1e88a5b6c9d2dbdba31e0f35f6a02461e7 and is unintentional if it should not be here.

Reply:
yep, the commit was reverted. You'll need to remove it here